packages/frontend/core/src/modules/dialogs/constant.ts

```diff
@@ -31,9 +43,13 @@ export type GLOBAL_DIALOG_SCHEMA = {
     workspaceMetadata?: WorkspaceMetadata | null;
     scrollAnchor?: string;
   }) => void;
-  'sign-in': (props: { server?: string; step?: 'sign-in' }) => void;
+  'sign-in': (props: { server?: string; step?: string }) => void;
   'change-password': (props: { server?: string }) => void;
   'verify-email': (props: { server?: string; changeEmail?: boolean }) => void;
+  'enable-cloud': (props: {
+    workspace?: Workspace;
+    options?: ConfirmEnableCloudOptions;
+  }) => void;
```

Comment:
```suggestion
  }) => boolean;
```
just return boolean to indicate whether the operation was completed successfully